doc/flame/components.md

```diff
@@ -911,13 +911,41 @@ void main() {
 
 ## TiledComponent
 
-Currently we have a very basic implementation of a Tiled component. This API uses the lib
-[tiled.dart](https://github.com/flame-engine/tiled.dart) to parse map files and render visible
-layers.
+Tiled is a free and open source, full-featured level and map editor for your platformer or
+RPG game. Currently we have an "in progress" implementation of a Tiled component. This API
+uses the lib [tiled.dart](https://github.com/flame-engine/tiled.dart) to parse map files and
+render visible layers using the performant `SpriteBatch` for each layer.
+
+Supported map types include: Orthogonal, Isometric, Hexagonal, and Staggered.
+
+Orthogonal | Hexagonal             |  Isomorphic
+:--:|:-------------------------:|:-------------------------:
+![An example of an orthogonal map](../images/orthogonal.png)|![An example of hexagonal map](../images/pointy_hex_even.png) |  ![An example of isomorphic map](../images/tile_stack_single_move.png)
 
 An example of how to use the API can be found
 [here](https://github.com/flame-engine/flame_tiled/tree/main/example).
 
+Once a `TiledComponent` is loaded, you can select any column of (x,y) tiles in a `tileStack` to
```

Comment:
I think that this could be it's own subsection

Reply:
Should flame_tiles have its own doc breakout?

Reply:
Created a sub-section for TileStack. Agreed we should have a whole doc section for flame_tiled.